src/database/database.cpp

```diff
@@ -0,0 +1,25 @@
+#include <iostream>
+
+#include <cstdint>
+#include <iostream>
+#include <vector>
+
+#include <bsoncxx/builder/basic/document.hpp>
+#include <bsoncxx/json.hpp>
+#include <mongocxx/client.hpp>
+#include <mongocxx/instance.hpp>
+#include <mongocxx/stdx.hpp>
+#include <mongocxx/uri.hpp>
+
+using bsoncxx::builder::basic::kvp;
+using bsoncxx::builder::basic::make_array;
+using bsoncxx::builder::basic::make_document;
+
+int main()
+{
+
+std::cout << "test";
```

Comment:
missing tabs. 

Maybe it is good idea to introduce clang-format to project. You can create task for that

Reply:
@KuWojtow, please take care of it